### PR TITLE
feat(bake): css, production build, dev separateSSRGraph=false

### PR DESCRIPTION
**Pull request by paperclover**

### CODE REVIEW

**Comment by robobun** [comment]
@paperdave, your commit 99025bb35c248339f97028d0b223eeb438ad816a is building: [`#4945`](https://buildkite.com/bun/bun/builds/4945)

File: src/bake/bun-framework-rsc/server.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import type { Bake } from "bun";
+import {} from "react";
```
Comment (dylan-conway) [inline src/bake/bun-framework-rsc/server.tsx]: delete

File: src/bundler/bundle_v2.zig [hunk 2]
```diff
@@ -943,33 +954,52 @@ pub const BundleV2 = struct {
...
-                    _ = try this.bundler.options.dev_server.?.server_graph.insertStaleExtra(resolved.path_pair.primary.text, false, true, route_index);
+                        try this.graph.entry_points.append(this.graph.allocator, Index.source(source_index));
+
+                        if (entry_point.route_index.unwrap()) |route_index| {
+                            _ = try this.bundler.options.dev_server.?.server_graph.insertStaleExtra(resolved.path_pair.primary.text, false, true, route_index);
+                        }
+
+                        if (entry_point.css) {
+                            try data.css_data.putNoClobber(this.graph.allocator, Index.init(source_index), .{ .imported_on_server = false });
+                        }
```
Comment (dylan-conway) [inline src/bundler/bundle_v2.zig:1008]: Should this block execute when `enqueueItem` returns null?

Comment (paperclover) [reply src/bundler/bundle_v2.zig:1008]: it shouldnt happen. a better thing to have would be `orelse @panic("")` continue is fine.

File: src/bundler/bundle_v2.zig [hunk 3]
```diff
@@ -8337,7 +8421,7 @@ pub const LinkerContext = struct {
                     &buffer_writer,
                     bun.css.PrinterOptions{
                         // TODO: make this more configurable
-                        .minify = c.options.minify_whitespace or c.options.minify_syntax or c.options.minify_identifiers,
+                        .minify = c.options.minify_whitespace,
```
Comment (dylan-conway) [inline src/bundler/bundle_v2.zig:8509]: `minify_syntax` and `minify_identifiers` don't apply?

Comment (dylan-conway) [reply src/bundler/bundle_v2.zig:8509]: oh, TODO, nvm

**Review by dylan-conway — APPROVED** [review]

File: .vscode/launch.json [hunk 4]
```diff
@@ -166,8 +166,8 @@
       "request": "launch",
       "name": "bun run [file] (fast)",
       "program": "${workspaceFolder}/build/debug/bun-debug",
-      "args": ["run", "${fileBasename}"],
-      "cwd": "${fileDirname}",
+      "args": ["build", "--app", "--static"],
+      "cwd": "/Users/dave/code/sample",
```
Comment (Jarred-Sumner) [inline .vscode/launch.json]: dave

**Pull request merged**